.convcommit.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is this file used for something?  Let's take it out if not.

Reply:
That's actually can be used by github actions.